Python_Visualization_Tutorial.Rmd

```diff
@@ -0,0 +1,7 @@
+# Python Visualization Tutorial
+
+Zixiang Tang(zt2292) & Chenxi Jiang(cj2706)
+
+Our task is to create a Jupyter notebook plot tutorial. This tutorial mainly focuses on plotting some commonly used graphs for visualization purposes. This tutorial includes most of the graphs we have learned in EDAV classes.
+
+Tutorial can be found via https://github.com/shawntzx/Python_Visualizaiton_Tutorial-EDAV_CC_Group15-.git
```

Comment:
Links to internal files must contain resources/<project_name>/ in the path, such as: ![Test Photo](resources/sample_project/election.jpg)

Reply:
I am sorry I don't quite get it. I couldn't see the photo. 
Does it mean that I need to put all my work in a folder under resources?

Reply:
Apologies for the delay in response, I believe you need to create a resources file under the same repo and save the tutorial there, then include a link to that file instead of the current link you included.

Reply:
I made the changes. Can you help me check if I am doing this correctly?

Reply:
Hello, 

I believe the changes are correct. However, you did not give write access to your files and therefore I cannot merge your document. please correct this.

Reply:
I just invited you to my repo. Is this the one you need access to?

Reply:
@jtr13 Hello professor, I still can't access the bookdown file since it requires me to close his work to do so. Could you let me know how I can solve this?